src/io/flutter/run/bazel/BazelFields.java

```diff
@@ -269,6 +274,12 @@ GeneralCommandLine getLaunchCommand(@NotNull Project project,
       commandLine.addParameter(additionalArgsTokenizer.nextToken());
     }
 
+    final String disableBazelHotRestartParam = "--no-enable-google3-hot-reload";
+    if (!FlutterSettings.getInstance().isEnableBazelHotRestart() &&
+        !StringUtil.notNullize(additionalArgs).contains(disableBazelHotRestartParam)) {
```

Comment:
Should I add a message or notification if this flag is in additional args? Could be useful if we want users to stop managing this themselves so we can switch over to default enabled when it's working more consistently in the future.

Reply:
Yes that is a good idea.